foundation/mobilecoinfoundation/attestation/v1/attest.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto3";
+
+// The APIs required to interact with the `AttestService`.
```

Comment:
per https://docs.buf.build/bsr/documentation#package-documentation, package documentation goes in the alphabetically first module prior to the package directive